IFComp/root/src/about/file_formats.tt

```diff
@@ -61,9 +61,15 @@
 
 <h3 id="adrift">ADRIFT games</h3>
 
-<p><strong>File extension:</strong> <code>.blorb</code></p>
+<p><strong>File extensions:</strong> <code>.taf</code>, <code>.blorb</code></p>
 
-<p>Games made with <a href="http://www.adrift.co">ADRIFT</a> also have a "blorb" file extension, just like <a href="#inform">games made with Inform</a>.</p>
+<p><a href="http://www.adrift.co">ADRIFT</a> is another system for creating parser-based IF. Like TADS and Inform, playing its games requires the use of <a href="http://www.adrift.co/files/ADRIFT5r.zip">a separate interpreter ("runner") program</a>, available for Windows.</p>
+
+<p>(Linux/Mac users: An alternate Linux/Mono interpreter is also available on the <a href="http://www.adrift.co/download">ADRIFT downloads page</a>, but current reports as of 2018 indicate that it doesn't work.)</p>
```

Comment:
Let's please remove this line. I'd rather not add instructions at all if they need to be tempered with "However, it doesn't work." :)

Reply:
I've done it, but I do think it is/was useful information that the Linux build doesn't work (so don't try).